akka-docs/src/main/paradox/cluster-client.md

```diff
@@ -23,23 +23,23 @@ to `remote` or `cluster` when using
 the cluster client.
 
 The receptionist is supposed to be started on all nodes, or all nodes with specified role,
-in the cluster. The receptionist can be started with the `ClusterClientReceptionist` extension
+in the cluster. The receptionist can be started with the @scala[@scaladoc[`ClusterClientReceptionist`](akka.cluster.client.ClusterClientReceptionist)]@java[@javadoc[`ClusterClientReceptionist`](akka.cluster.client.ClusterClientReceptionist)] extension
 or as an ordinary actor.
 
-You can send messages via the `ClusterClient` to any actor in the cluster that is registered
-in the `DistributedPubSubMediator` used by the `ClusterReceptionist`.
-The `ClusterClientReceptionist` provides methods for registration of actors that
+You can send messages via the @scala[@scaladoc[`ClusterClient`](akka.cluster.client.ClusterClient)]@java[@javadoc[`ClusterClient`](akka.cluster.client.ClusterClient)] to any actor in the cluster that is registered
+in the @scala[@scaladoc[`DistributedPubSubMediator`](akka.cluster.pubsub.DistributedPubSubMediator)]@java[@javadoc[`DistributedPubSubMediator`](akka.cluster.pubsub.DistributedPubSubMediator)] used by the @scala[@scaladoc[`ClusterReceptionist`](akka.cluster.client.ClusterReceptionist)]@java[@javadoc[`ClusterReceptionist`](akka.cluster.client.ClusterReceptionist)].
+The @scala[@scaladoc[`ClusterClientReceptionist`](akka.cluster.client.ClusterClientReceptionist)]@java[@javadoc[`ClusterClientReceptionist`](akka.cluster.client.ClusterClientReceptionist)] provides methods for registration of actors that
 should be reachable from the client. Messages are wrapped in `ClusterClient.Send`,
-`ClusterClient.SendToAll` or `ClusterClient.Publish`.
+@scala[@scaladoc[`ClusterClient.SendToAll`](akka.cluster.client.ClusterClient$)]@java[`ClusterClient.SendToAll`] or @scala[@scaladoc[`ClusterClient.Publish`](akka.cluster.client.ClusterClient$)]@java[`ClusterClient.Publish`].
```

Comment:
### scala
scaladoc is linked to:
https://doc.akka.io/api/akka/2.5/akka/cluster/client/ClusterClient$

As the URL for `SendToAll` includes the return type, and messy (also the part after `#`might be subject to change in future?).
https://doc.akka.io/api/akka/2.5/akka/cluster/client/ClusterClient$.html#SendextendsProductwithSerializable

### java
javadoc link is not added as there is no way to get it work. 
The correct URL is:
https://doc.akka.io/japi/akka/2.5/akka/cluster/client/ClusterClient.SendToAll.html,

but `@javadoc[...](akka.cluster.client.ClusterClient.Publish)`  will be linked to 
.....client/ClusterClient/SendToAll.html (`/` after ClusterClient) 
which doesn't exist.